src/eterna/net/GameClient.ts

```diff
@@ -18,7 +19,10 @@ interface VoteDataWrapper {
     votes: VoteData[];
 }
 
-type Params = Record<string, string | number | boolean | PuzzleJSON | RankScrollData | null | undefined >;
```

Comment:
We have a es-lint rule that doesn't allow any line to be longer than 120 characters. Do you have a preferred way to break these sorts of expressions into two lines? In general I wasn't sure if we had an Eterna best practice for formatting multi-line expressions (this includes conditionals).

Reply:
As a general rule my preference is first to break at braces (`()`, `{}`, `[]`, `<>`), then commas, then conditionals (with a conditional like `||` going on the following line). It's somewhat specific to the situation though, so it's hard to describe hard and fast rules. In this case, I'd probably do something like:
```
type Params = Record<
    string,
    string | number | boolean | PuzzleJSON | RankScrollData | null | undefined
>;
 ```

Reply:
This does seem awfully suspicious that _all of these_ are in a single type though.

Reply:
Made this change in the new commit. Regarding the suspiciousness, I agree there, but don't know what code it covers to suggest a fix

Reply:
`Params` is, painfully, used in a lot of places. It is a barely-more-constrained `any` that's usually used to represent json returned from some kind of query, I think.

It could be done a lot better.